tools/pubsys/policies/ssm/defaults.toml

```diff
@@ -0,0 +1,7 @@
+[[parameter]]
+name = "{variant}/{arch}/{image_version}/image_id"
+value = "{image_id}"
+
+[[parameter]]
```

Comment:
Don't we have one of these for `current`? Should that be in the defaults.toml as well?

Reply:
For `cargo ssm`, the `image_version` in the template is replaced by the version you're building, and for `promote-ssm`, by the version you specify with `-e SSM_TARGET=`.  So when we want to publish `latest`, we'd do `cargo make promote-ssm -e SSM_TARGET=latest`.  In our release process we're planning to promote once from the full version (e.g. `0.5.0-abcdef`) to the short version (`0.5.0`) and again to `latest`.